zerver/management/commands/delete_old_unclaimed_attachments.py

```diff
@@ -42,10 +53,39 @@ def handle(self, *args: Any, **options: Any) -> None:
                 f"* {old_archived_attachment.file_name} created at {old_archived_attachment.create_time}"
             )
 
-        print("")
+        if options["for_real"]:
+            do_delete_old_unclaimed_attachments(delta_weeks)
+            print("")
+            print("Unclaimed files deleted.")
+
+        if options["clean_up_storage"]:
+            print("")
+            self.clean_storage_backends(dry_run=not options["for_real"])
+
         if not options["for_real"]:
+            print("")
             raise CommandError("This was a dry run. Pass -f to actually delete.")
 
-        do_delete_old_unclaimed_attachments(delta_weeks)
-        print("")
-        print("Unclaimed files deleted.")
+    def clean_storage_backends(self, dry_run: bool = True) -> None:
```

Comment:
Why is this called `..._backends`? That makes it sounds like it may have something to do with multiple storage backends - perhaps we should just call it `clean_file_storage` or such?

Reply:
The class is `ZulipUploadBackend` and the global is named `upload_backend`, so it feels useful to have that word somewhere in here.  But I agree that there's only one, so the plural isn't justified -- would `clean_attachment_upload_backend` see reasonable?

Reply:
Yeah sounds good to me